src/forms/finalFormComponent.jsx

```diff
@@ -100,17 +100,39 @@ const inputTypeProp = (props, propName) => {
   return undefined;
 };
 
+const optionsPropType = (props, propName, componentName) => {
+  if (props.componentType === 'select') {
+    if (!Array.isArray(props.options)) {
```

Comment:
all this custom validation code is needed because it applies only to `componentType === 'select'` ?

Reply:
i plan to expand it. In select component, you can specify the name of the key and value attribute of options. A put there in preparation in the future. I wanted to include it into this PR, but for some reason it wasn't working properly. Should i remove it?

Reply:
No need to remove it. I am just trying to understand it ;-)